hybrasyl/Dialogs.cs

```diff
@@ -21,22 +21,19 @@
  */
 
 using Hybrasyl.Objects;
-using log4net;
-using System;

```

Comment:
pls readd `System`, unexpected results can occur when removing.

Reply:
lol wut

Reply:
in rare occasions, even where something isn't referenced directly from the base `System` class, unexpected results can occur due to the namespace not being included. it's a very old .net bug. not sure if it applies to core still

Reply:
fixed